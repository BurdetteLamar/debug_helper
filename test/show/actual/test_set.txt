---
Set (message='test_set'):
  Set#size: 2
  Element 0: Fixnum 14
  Element 1: Fixnum 22
