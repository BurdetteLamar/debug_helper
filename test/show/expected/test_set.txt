---
Set (message='test_set' size=2):
  Element 0: Fixnum 14
  Element 1: Fixnum 22
